build.sbt

```diff
@@ -54,7 +54,7 @@ val dep = {
 }
 
 val verifyDeps = Seq(
-  "org.scala-lang" % "scala-library" sha1 "dd235d04037dc6f4b6090257872dd35359a563ce",
+  "org.scala-lang" % "scala-library" sha1 "4861b00d921952f2dc9a024198bc72590fd7dc5e",
```

Comment:
So, what was the problem/solution?

Reply:
`sbt-verify`: locally the last build step ("additional compilation & dist") fails, regardless of Scala version. No solution for that but I think we should decouple the scala version upgrade from whatever `sbt-verify` does. The above changes are just to take into account the modified `2.12.5` scala jars.

Reply:
Yes, it fails locally for me too :(

> decouple the scala version upgrade from whatever sbt-verify does

How can we do that? Unless you mean removing `sbt-verify` (or treating the failed checks as warnings) which I would be happy with 😁